### PR TITLE
fix: add privacy and termsofuse page

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: share-now/tabs/src/Privacy.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
```
Comment (adashen) [inline share-now/tabs/src/Privacy.tsx:1]: file header?

File: share-now/tabs/src/TermsOfUse.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
```
Comment (adashen) [inline share-now/tabs/src/TermsOfUse.tsx:1]: file header

File: share-now/tabs/src/Privacy.tsx (hunk 1 above)
Comment (KennethBWSong) [reply share-now/tabs/src/Privacy.tsx:4]: Fixed. Thanks!

File: share-now/tabs/src/TermsOfUse.tsx (hunk 2 above)
Comment (KennethBWSong) [reply share-now/tabs/src/TermsOfUse.tsx:4]: Fixed. Thanks!

**Review by adashen — APPROVED** [review]

**Pull request merged**